packages/KTable/KTable.vue

```diff
@@ -184,6 +217,19 @@ export default {
 
 table.k-table {
   border-collapse: collapse;
+
+  &.side-border {
```

Comment:
I think its weird we have these two .k-table definitions. Lets combine everything into one nested css obj. This is left over formatting from before scss I think.

Reply:
re: my comment above for the border-spacing I think it was originally just to add that. it seems we always want `border-collapse: separate` now though, but that will actually remove the borders from each row, which will also affect KM as well and everything that uses `KTable` if we are fine with that

Reply:
The default should be collapse. And when `side-border` is applied its `separate`. so no need to remove the border as I mentioned in another comment, that will be done automatically. For KM we will add `hasBorder="false"` by default so they retain the borders

Reply:
yeah but for `rowAttrs` we also need the border to be separate too so should we just repeat some styles there? the reason I had `sideBorder` on to require rowAttrs was this reason

Reply:
Why do you need it separate for rowAttrs? technically rowAttrs has nothing to do with the border. It’s just a helper function to style based on row data.

Reply:
ok, but in the example we would want to have different classes on each row and each border to give it a different color. So in that case, we have to make the border collapse separate. You're right that `rowAttrs` could be anything, not just related to borders. But in the case we do use it for borders which is the example case, I think we have to add this in. So I guess I'll just repeat it just for the example then.

Reply:
I think it’s we’re just misunderstanding each other. You had said in order for `rowAttrs` to be applied you needed `sideBorder` and my point was they aren’t related. Just in the KHCP use case we’re using one to style the other. Happy to discuss over a call 🤓